drivers/bridge/bridge.go

```diff
@@ -300,7 +300,8 @@ func (n *bridgeNetwork) isolateNetwork(others []*bridgeNetwork, enable bool) err
 			}
 		}
 
-		if thisV6 != nil && otherV6 != nil && !types.CompareIPNet(thisV6, otherV6) {
+		// TODO: Enable when full ipv6 table infrastructure is available
+		if false && thisV6 != nil && otherV6 != nil && !types.CompareIPNet(thisV6, otherV6) {
```

Comment:
Yes, that would be cleaner.
Although this PR's change is meant to unblock multiple network creation which is now impossible if  user assigns a V6 subnets.

Let me check if I can get the interface based isolation rule in place and working soon.